src/utils/Logger.ts

```diff
@@ -0,0 +1,30 @@
+import { LogInterface } from "../interfaces/LogInterface";
+
+export class Logger implements LogInterface {
+  public debug(msg: string, ...supportingData: any[]): void {
```

Comment:
We don't need to use the `public` keyword in classes. In JavaScript/TypeScript this is the default behaviour

Reply:
`doActive` is still using this.